### PR TITLE
Anti-spam for discord invite links, update deps, and other updates

### PR DESCRIPTION
**Pull request by D0cR3d**
**General:**

- Logs when user joined server and discord when user joins/parts.
- Updated deps

**VCM:**

- Removed duplicate voice channel names (Cayde, Zavala, Ikora)
- Added additional names ('Baby Dog' #135, War Beast names from Pleasure Gardens in raid, Calus), 

**Mute:**

- Refactored Action:Mute to take advantage of Promises and set the role first.

**Clear:**

- Added alias of `sweep` because obvious reasons. Future update can allow you to `.sweep @user` to remove only that persons messages.

**Warn:**

- Updated the 'clan' shortcode to `Please, no clan advertising as per Rule 3. If you'd like to advertise your clan, we recommend the weekly thread on r/Fireteams or r/DTG's own Team-Up Tuesday. Thank you!`

**Events:**

- Processes all messages, and if there is a Discord Invite Link posts by someone who does NOT possess the 'The Vanguard' or 'Moderators' role, it will remove the message, warn the user, log in database, and log in Sweeper Logs channel. (#100)
- No longer removes a reaction created by Charlemagne bot. This caused issues with their bot not being able to allow users to switch platforms in the stats channel.

### CODE REVIEW

File: src/util/lib/listeners/Events.ts [hunk 1]
```diff
@@ -33,7 +35,8 @@ export class Events {
 		if (user.id === this._client.user.id)
 			return;
 
-		if (user.bot && user.id !== this._client.user.id)
+		// 296023718839451649 == @Charlemagne#3214
+		if (user.id !== '296023718839451649' || user.bot && user.id !== this._client.user.id)
```
Comment (weylin) [inline src/util/lib/listeners/Events.ts]: Just noting that we are targeting a known bot that has a an ID that is unlikely to change. Should it change or we extend this to more bots we should move to using a the loadable configuration file instead.

File: src/util/Constants.ts [hunk 2]
```diff
@@ -81,6 +83,6 @@ Constants.FWCEmoji = '<:fwc:247889245337944064>';
...
...SABER' , 'Saint-14' , 'Sardok' , 'Sardon' , 'Sathona' , 'Saviks' , 'Sayeth' , 'Sedia' , 'Segoth' , 'Sekrion' , 'Sepiks' , 'Shaxx' , 'Shirazi' , 'Shiro-4' , 'Shuro' , 'Shvubi' , 'Silimar' , 'Simiks' , 'Skoriks' , 'Skorri' , 'Skriviks' , 'Skyburners' , 'Solkis' , 'Starcutters' , 'Sundaresh' , 'Suros' , 'Sweeper Bot' , 'Swiftling' , 'Sylok' , 'Taeko' , 'Taishibethi' , 'Taniks' , 'Taox' , 'Tarlowe' , 'Teben' , 'Techeun' , 'Techeuns' , 'Telthor' , 'Tescan' , 'Tevis' , 'Thalnok' , 'The Speaker' , 'The Traveler' , 'Theosyion' , 'Thuria' , 'Timur' , 'Tinette' , 'Toland' , 'Tover' , 'Trenn' , 'Tubach' , 'Tuyet' , 'Uldren Sov' , 'Urrox' , 'Urzok' , 'Uzoma' , 'Valus' , 'Vekis' , 'Veliniks' , 'Velor' , 'Venj' , 'Vestan' , 'Vestian' , 'Virixas' , 'Vorlog' , 'Warmind' , 'Warminds' , 'Warpriest' , 'Warsat' , 'Wei Ning' , 'Weksis' , 'Wintership' , 'WorldRender' , 'Wormfood' , 'Xander 99-40' , 'Xivu' , 'Xol' , 'Xur' , 'Xyor' , 'Yavek' , 'Zarin' , 'Zhalo' , 'Zire' , 'Zydron' , 'Zyre' , 'Zyrok' , 'Fallen' , 'Taken' , 'Cabal' , 'Vex' , 'Harpy' , 'Dreg' , 'Vandal' , 'Hunter' , 'Titan' , 'Warlock' , 'Ogre' , 'Acolyte' , 'Thrall' , 'M\'Orn' , 'Ta\'Aurc' , 'B\'Ael' , 'Aru\'Un' , 'Tho\'Ourg' , 'Va\'Ase' ];
```
Comment (weylin) [inline src/util/Constants.ts]: Nitpicking at the line length.

Comment (katagatame) [reply src/util/Constants.ts]: TBH, we could move this to a json file.

File: package.json [hunk 3]
```diff
@@ -29,10 +29,10 @@
     "fuzzy": "^0.1.3",
     "moment": "^2.18.1",
     "node-schedule": "^1.2.4",
-    "pg": "7.1.2",
+    "pg": "7.3.0",
```
Comment (katagatame) [inline package.json]: 7.3.0 is an insecure dependency.

We still need to target 7.1.2.

File: src/config.example.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
 	"owner": [" owner id "],
 	"name": "Sweeper Bot",
 	"version": "0.1.0",
-	"status": "Try .help",
+	"status": "Try .help | DMing me DMs the mods",
```
Comment (katagatame) [inline src/config.example.json]: This might be a little too long.

**Pull request merged**